platform/core/Cargo.toml

```diff
@@ -1,37 +1,7 @@
-[package]
-name = "polypod-core"
-version = "0.1.0"
-edition = "2021"
+[workspace]
+members = ["lib", "ffi", "crates/*"]
```

Comment:
For the sake of clarity, will it be an improvement to list all the libraries in `crates/`?

Reply:
Could be, but my intent is for `crates/` to contain the subcomponents, while lib and ffi are the high level crates, which represent a composition of `crates/`.
All of the new functionality should be added into `crates/`. That being said, I don't expect other crates to be added at the same level as `lib` and `ffi`.